app/assets/stylesheets/partials/_results.scss

```diff
@@ -7,17 +7,16 @@
   background-color: $white-t;
   padding: 15px;
 
+  @media (max-width: $bp-screen-md) {
+    padding: 0px;
+  }
   .no-results {
     margin-left: -15px;
   }
 
   .no-results + .ask-us {
     margin-top: 0rem;
   }
-
-  #results {
```

Comment:
I really like this change. I'm sure it'll be controversial, but this is the way.

Reply:
No controversy so far! Darcy likes it, too.